docs/usage/configuration-options.md

```diff
@@ -1912,6 +1912,8 @@ Normally when you set `rebaseWhen=auto` Renovate rebases any branch that's behin
 This behavior is no longer guaranteed when you enable `platformAutomerge` because the platform might automerge a branch which is not up-to-date.
 For example, GitHub might automerge a Renovate branch even if it's behind the base branch at the time.
 
+Please check platform specific docs for version requirements.
```

Comment:
Some of our platform docs do not mention what versions users should use for "full support for all Renovate features":

| Platform docs for | Mentions fully supported version? | Has section "supported versions" |
| --- | --- | --- |
| Azure | no | no |
| Bitbucket Cloud | no | no |
| Bitbucket Server | We don't support EOL versions | yes |
| Gitea | no, only mentions version for a single feature | no |
| GitHub and GitHub Enterprise Server | mentions GHES token version | no |
| GitLab | Mentions server version dependent features | no |

Reply:
I have opened an issue to discuss this problem further:

- #16937